prime-router/.java-version

```diff
@@ -1 +1 @@
-11
+17
```

Comment:
Might want to clear this with platform first? 
<img width="1089" alt="Screenshot 2023-09-14 at 5 14 30 PM" src="https://github.com/CDCgov/prime-reportstream/assets/516877/71b17687-5bbc-4116-aaa0-778e1366ebe2">

Reply:
@thetaurean do you want me to remove this from my PR?

Reply:
Yes please, I have a small PR addressing this as well. We landed on `17.0` instead of `17` after testing.

Thanks for checking in @penny-lischer and thanks @etanb for the callout.